ocean/performance_testing/performance_test.py

```diff
@@ -165,6 +165,6 @@
     figurenamepath = "figures_performance/" + resolution_name + '_' + \
         machine_long_name + '_' + str(nprocs_max) + "_" + timenow + ".png"
     plt.savefig(figurenamepath)
-    i = i / 2
+    i = int( i / 2 )
```

Comment:
The preferred way to do this is:
```suggestion
    i = i // 2
```